rancher2/schema_node_template_hetzner.go

```diff
@@ -13,6 +13,7 @@ const (
 type hetznerConfig struct {
 	APIToken          string   `json:"apiToken,omitempty" yaml:"apiToken,omitempty"`
 	Image             string   `json:"image,omitempty" yaml:"image,omitempty"`
+	ServerLabels      []string `json:"serverLabels,omitempty" yaml:"serverLabels,omitempty"`
```

Comment:
The `ServerLabels` field type doesn't match what is defined at [JonasProgrammer/docker-machine-driver-hetzner](https://github.com/JonasProgrammer/docker-machine-driver-hetzner/blob/3.7.0/driver.go#L47).  It should be `map[string]string` instead of `[]string`